app/controllers/application_controller.rb

```diff
@@ -1,2 +1,6 @@
 class ApplicationController < ActionController::Base
+  def current_user
+    @user = User.first
+    @user
```

Comment:
- [ ] we don't need to assign the user to a new variable so User.first along will be fine.